.gitignore

```diff
@@ -1,13 +1,13 @@
-/.bundle/
-/.yardoc
-/Gemfile.lock
-/_yardoc/
-/coverage/
-/doc/
-/pkg/
-/spec/reports/
-/tmp/
-/html/
-/lib/gemstash/man/
+.bundle/
+.yardoc
+Gemfile.lock
+_yardoc/
+coverage/
+doc/
+pkg/
+spec/reports/
+tmp/
+html/
+lib/gemstash/man/
```

Comment:
https://git-scm.com/docs/gitignore seems to describe the initial slash, as well.

That is, in the last example.